lsp-mode.el

```diff
@@ -9577,6 +9570,18 @@ modifying window sizes."
 (defun lsp--update-inlay-hints ()
   (lsp-update-inlay-hints (window-start) (window-end nil t)))
 
+(defun lsp--label-from-inlay-hints-response (parts)
+  "Returns a string label built from an array of
+InlayHintLabelParts or the argument itself if it's already a
+string."
+  (cond
+   ((stringp parts) parts)
+   ((arrayp parts)
+    (mapconcat (lambda (part)
```

Comment:
nit: you can use -lambda and destructure directly in the lambda params secion

Reply:
Thanks for the tip, I'm not very proficient in Emacs Lisp.